src/entry_point_win.cpp

```diff
@@ -211,14 +214,21 @@ int WINAPI wWinMain(HINSTANCE /*hInstance*/, HINSTANCE /*hPrevInstance*/, LPWSTR
 		text::load_bmfonts(game_state);
 		ui::populate_definitions_map(game_state);
 
-		std::thread update_thread([&]() { game_state.game_loop(); });
-
-		// entire game runs during this line
-		window::create_window(game_state,
-				window::creation_parameters{1024, 780, window::window_state::maximized, game_state.user_settings.prefer_fullscreen});
+		if(headless) {
+			game_state.actual_game_speed = 6;
+			game_state.ui_pause.store(false, std::memory_order::release);
+			game_state.mode = sys::game_mode_type::in_game;
+			game_state.local_player_nation = dcon::nation_id{};
+			game_state.game_loop();
```

Comment:
but how do you quit?

Reply:
VTune can stop/kill the process, which is sufficient to quit
also the user can manually Ctrl-C the process or kill it
requiring input to quit may not be a good idea since we dont provide a console per se, and this is more meant to be a "alice runs as a children of another process who controls when it starts/stops/etc"